test/jdk/java/foreign/TestBufferStackStress2.java

```diff
@@ -23,6 +23,10 @@
 
 /*
  * @test
+ *
+ * If the VM does not have continuations, then no VTs can ever be scheduled
```

Comment:
Nit - jtreg allows for a `@comment` tag in the test definition https://openjdk.org/jtreg/tag-spec.html. It would be good to prefix this line with `@comment`.